modules/portal/portal-api/src/main/java/com/enonic/xp/portal/UniversalApiHandlerRegistry.java

```diff
@@ -0,0 +1,14 @@
+package com.enonic.xp.portal;
+
+import java.util.List;
+
+import com.enonic.xp.api.ApiDescriptor;
+import com.enonic.xp.page.DescriptorKey;
+import com.enonic.xp.web.universalapi.UniversalApiHandlerWrapper;
+
+public interface UniversalApiHandlerRegistry
+{
+    UniversalApiHandlerWrapper getApiHandler( DescriptorKey key );
```

Comment:
"Wrapper" is not a good name for api
Alos, one is in web, another is in portal - something is wrong